build

```diff
@@ -19,5 +19,16 @@ export GOBIN=${PWD}/bin/${GOARCH}
 export GOPATH=${PWD}/gopath
 export CGO_ENABLED=1
 
+if [ "${ACTION-BUILD}" == "NOBUILD" ]; then
```

Comment:
build script with a NOBUILD option smells.

The actual problem you're trying to solve is that there are now test binaries. This is where using a Makefile becomes a better choice than using this script.